eacl/target.go

```diff
@@ -58,6 +60,27 @@ func (t *Target) SetBinaryKeys(keys [][]byte) {
 	t.keys = keys
 }
 
+// Accounts returns list of accounts to identify target subject in a binary format.
+//
+// Each element of the resulting slice is a 25 byte account.
```

Comment:
user can do `t.SetAccounts([][]byte{[]byte("Hello, world!")})` making this statement false. I would emphasize that this is the expected length, but to decode it you need to explicitly check it

Reply:
Added